src/main/java/com/google/sps/servlets/AnalyzedVideo.java

```diff
@@ -61,8 +63,9 @@ public static String getRandomTitle() {
     return titlesList.get(rand.nextInt(4));
   }
 
-  public static double getRandomSentiment() {
-    return sentimentList.get(rand.nextInt(4));
+  public static double getSentiment(String text) throws IOException {
+    Sentiment sentimentFromText = analyze.analyzeSentimentText(text);
+    return sentimentFromText.getScore();
   }
```

Comment:
My suggestion would be to not do any work inside the model (AnalyzedVideo). Instead, call this in `DataServlet` to get the sentiment score before setting it in the model.